tests/test_build_images.py

```diff
@@ -506,3 +520,10 @@ def test_architecture(self):
         Test architecture of this build image
         """
         self.assertTrue(self.is_architecture("aarch64"))
+
+    def test_packages(self):
```

Comment:
Can we rename the method something with `go` in case we need to  add SDKs for other languages which can be used in `provided.al2`?